examples/next-13-app-router-graphql/app/providers.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
```

Comment:
@z0al this is the part i was talking about, without this `use client` we are getting an error even though the provider has it's own directive